### PR TITLE
Upgraded prettier and eslint plugins, removed RN eslint-config from package.json

### PR DESCRIPTION
**Pull request by vbudziak-revolve**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,16 +25,15 @@
   "license": "ISC",
   "homepage": "https://bitbucket.org/revolve-it/eslint-config-revolve#readme",
   "dependencies": {
-    "@react-native-community/eslint-config": "^3.2.0",
```
Comment (vbudziak-revolve) [inline package.json:28]: RN eslint config is removed from deps because it is required from react native by default

**Review by PrzemoRevolve — APPROVED** [review]
:heavy_check_mark: if you test that it works :grin:

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-revolve",
-  "version": "3.0.0",
+  "version": "4.0.0-beta.1",
```
Comment (jganczorz-revolve) [inline package.json]: I do not feel great about pushing `beta` version to the `master` branch

**Review by jganczorz-revolve — COMMENTED** [review]
I'll try to use this config in my current project and checks if there is no problems

File: package.json (hunk 2 above)
Comment (vbudziak-revolve) [reply package.json]: ok i will change it to 4.0.0 then

**Pull request merged**